.docker/docker-compose.development.yml

```diff
@@ -14,6 +14,7 @@ services:
       DB_HOSTNAME: td_mongodb
       DB_PORT: 26900
       TEST_DB_HOSTNAME: test_mongodb
+      FRONTEND_URL: www.frontend.com
```

Comment:
frontend.com?

Reply:
Yes, it has to be set depending on where the frontend used is running

Reply:
Will it be `www.www.frontend.com` in this example since `www.` is added in the code as well? Maybe remove `www.` from where the string is formatted with the env variable?

Reply:
Indeed, well spotted! I was certain I fixed that, must have forgotten to commit the changes 😅